### DISCUSSION TITLE
laf,logo

### DISCUSSION THREAD

**Discussion by hsfanrice** [labring/laf#356] [Generals]
![logo](https://user-images.githubusercontent.com/50478918/193009154-bad75970-e859-4a0a-a6fb-1c0a243d9069.png)

logo说明：外圆表示平台或者生态意思哈，缺口呢开放意思，整个外圆也像加载中哈，表示持续更新迭代不停滞不前意思哈。
两个眼睛Open source and open开源开放，
翅膀D形式，表示开发者developer意思哈。
如有雷同纯属巧合，谢谢观赏！

**Comment by fanux** [comment]
寓意很丰富 哈哈
